frontend/public/components/command-line-tools.tsx

```diff
@@ -54,44 +54,15 @@ const CommandLineTools: React.FC<CommandLineToolsProps> = ({ obj }) => {
         <h1 className="co-m-pane__heading">
           <div className="co-m-pane__name">{title}</div>
         </h1>
-        <h2 className="co-section-heading">oc - OpenShift Command Line Interface (CLI)</h2>
-        <p>
-          With the OpenShift command line interface, you can create applications and manage
-          OpenShift projects from a terminal.
-        </p>
-        <p>
-          The oc binary offers the same capabilities as the kubectl binary, but it is further
-          extended to natively support OpenShift Container Platform features.
-        </p>
-        <p>
-          <ExternalLink href={OC_DOWNLOAD_LINK} text="Download oc" />
-          {(window as any).SERVER_FLAGS.requestTokenURL && (
-            <React.Fragment>
-              &nbsp;
-              <span className="co-action-divider" aria-hidden="true">
-                |
-              </span>
-              &nbsp;
-              <ExternalLink
-                href={(window as any).SERVER_FLAGS.requestTokenURL}
-                text="Copy Login Command"
-              />
-            </React.Fragment>
-          )}
-        </p>
-        <hr />
-        <h2 className="co-section-heading">odo - Developer-focused CLI for OpenShift</h2>
-        <p>
-          OpenShift Do (odo) is a fast, iterative, and straightforward CLI tool for developers who
-          write, build, and deploy applications on OpenShift.
-        </p>
-        <p>
-          odo abstracts away complex Kubernetes and OpenShift concepts, thus allowing developers to
-          focus on what is most important to them: code.
-        </p>
-        <p>
-          <ExternalLink href={ODO_DOWNLOAD_LINK} text="Download odo" />
-        </p>
+        {(window as any).SERVER_FLAGS.requestTokenURL && (
+          <React.Fragment>
```

Comment:
nit: Prefer the `<>` fragment shorthand

```suggestion
          <>
```